### ISSUE TITLE
2 rules conflict with each other

### ISSUE THREAD

**Issue by vladyslavstartsev** [slevomat/coding-standard#490]
If I  have enabled 2 rules
the `SlevomatCodingStandard.Arrays.TrailingArrayComma.MissingTrailingComma` and `Squiz.Arrays.ArrayDeclaration.NoComma` it will incorrectly make beautifying for the file
for example what it does with such file 
```php
<?php
//pre phpbcf
declare(strict_types=1);

return [
    'notifications'              => [
        'channel' => 'database'
    ]
];
```

error on cli 
![image](https://user-images.githubusercontent.com/17382248/45926637-2716b300-bf2e-11e8-9ab2-661ce575598b.png)

```php
<?php
//post phpbcf
declare(strict_types=1);

return [
    'notifications'              => [
        'channel' => 'database',,
    ],
];
```

see this 2 commas at the end? That't the problem. How can this be fixed? thank you!

**Comment by kukulich** [comment]
You should disable one rule.
